src/assets/js/languageSelector.js

```diff
@@ -15,7 +15,13 @@ var languageSelector = {
     
     $('.section').hide();
     this.showSection(languageParam);
-    $('code').hide();
+    
+    $('code').each(function(){
+      if (typeof $(this).attr('class') != 'undefined' && $(this).attr('class') != "hljs") {
```

Comment:
Nitpicking: 
 - `!=` shouldn't be `!==` ?
 - `"hljs"`: use single quote for string
 - You use the same selector 3 times (`$(this)`), maybe you can store the result in a variable to avoid unnecessary dom parsing